roxie/udplib/udptrr.cpp

```diff
@@ -690,40 +749,17 @@ class CReceiveManager : implements IReceiveManager, public CInterface
     typedef std::map<ruid_t, CMessageCollator*> uid_map;
     uid_map         collators;
     SpinLock collatorsLock; // protects access to collators map
-    // inflight is my best guess at how many packets may be sitting in socket buffers somewhere.
-    // Incremented when I am notified about packets having been sent, decremented as they are read off the socket.
-    std::atomic<int> inflight = {0};
 
     int free_slots()
```

Comment:
no longer called so delete - the comment on non-blocking isn't correct...

Reply:
removed